CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Support for C#'s `packages.*.config` lockfile type
 - `phylum firewall log` command to browse firewall activity log
 
+### Fixed
+
+- Gradle manifests used on Windows when legacy lockfiles exist
```

Comment:
Not sure I like the `legacy` phrasing here. Sounds a bit like we're using outdated lockfiles over updated manifests when realistically the entire project is "legacy".

Reply:
I can change it if desired. How about `Gradle manifests incorrectly retained on Windows`?

Reply:
It has been updated.